src/ConnectionConfig.tsx

```diff
@@ -36,12 +36,23 @@ export const ConnectionConfig: FC<ConnectionConfigProps> = (props: ConnectionCon
   if (profile === undefined) {
     profile = options.database;
   }
+  const tempCredsFeatureEnabled = config.featureToggles.awsDatasourcesTempCredentials;
 
   const settings = (window as any).grafanaBootData.settings;
   const awsAllowedAuthProviders = useMemo(
-    () => settings.awsAllowedAuthProviders ?? [AwsAuthType.Default, AwsAuthType.Keys, AwsAuthType.Credentials],
-    [settings.awsAllowedAuthProviders]
+    () => [
+      ...(
+        settings.awsAllowedAuthProviders ?? [
+          AwsAuthType.Default,
+          AwsAuthType.Keys,
+          AwsAuthType.Credentials,
+          AwsAuthType.GrafanaAssumeRole,
```

Comment:
on that note, do either of you think it needs to be made more obvious that this option is meant only for cloud? (since this would be the first time an AWS auth option is meant just for cloud, right?)

Reply:
Im ok with not putting it in the config for now, since we will probably add it in the docs under AWS authentication once it goes GA. @sarahzinger what do you think?